main.py

```diff
@@ -1,17 +1,198 @@
-"""
-This is a starter file to get you going. You may also include other files if you feel it's necessary.
+import ssl
 
-Make sure to follow the code convention described here:
-https://github.com/UWARG/computer-vision-python/blob/main/README.md#naming-and-typing-conventions
+import matplotlib.pyplot as plt
+import tensorflow as tf
 
-Hints:
-* The internet is your friend! Don't be afraid to search for tutorials/intros/etc.
-* We suggest using a convolutional neural network.
-* TensorFlow Keras has the CIFAR-10 dataset as a module, so you don't need to manually download and unpack it.
-"""
+ssl._create_default_https_context = ssl._create_unverified_context
 
-# Import whatever libraries/modules you need
 
-import numpy as np
+def load_data():
 
-# Your working code here
+    """
+    This function loads the data onto the training and testing variables
+
+    Furthermore, it one hot encodes the y test and train variables and 
+    normalizes the x train and test variables to be between 0 and 1
+
+    Returns
+    -------
+    Returns a tuple of training and testing values for x and y
+    """
+
+    # Loading data to develop the model
+    (xTrain, yTrain), (xTest, yTest) = tf.keras.datasets.cifar10.load_data()
+
+    # Normalizing data inputs to be between 0 and 1
+    xTrain = xTrain.astype("float32")
+    xTest = xTest.astype("float32")
+    xTrain = xTrain/255.0
+    xTest = xTest/255.0
+
+    # One hot encoding target values
+    yTrain = tf.keras.utils.to_categorical(yTrain)
+    yTest = tf.keras.utils.to_categorical(yTest)
+
+    return xTrain, xTest, yTrain, yTest
+
+
+def create_model(xTrain, yTest):
```

Comment:
just a small thing here, you can only pass in the required shape for xTrain and yTest instead of passing all of them

Reply:
other wise all good here, good job, this is beautiful code